### PR TITLE
add ExtractDatahubExportCountryHistory task

### PR DESCRIPTION
**Pull request by dvoong**

### CODE REVIEW

File: app/db/models/external.py [hunk 1]
```diff
@@ -50,6 +50,20 @@ class DatahubContact(BaseModel):
...
+
+    __tablename__ = 'datahub_export_country_history'
+    __table_args__ = {'schema': 'public'}
+
+    company_id = _col(UUID(as_uuid=True))
+    country = _col(_text)
+    country_iso_alpha2_code = _col(_text)
+    history_date = _col(_dt)
+    history_type = _col(_text)
+    id = _col(UUID(as_uuid=True), primary_key=True)
```
Comment (marcuspp) [inline app/db/models/external.py]: This isn't consistent with all the other models. The PK is a UUID here and int everywhere else. Could this be made to be changed to be consistent?

File: app/etl/tasks/datahub_country_export.py [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
+        d.id::text as source_id,
+        history_date::timestamp as timestamp
+
+    from {DatahubExportCountryHistory.get_fq_table_name()} d
+        left join {DITCountryTerritoryRegister.get_fq_table_name()} c
+            on d.country_iso_alpha2_code = c.country_iso_alpha2_code
+        left join {StandardisedCountries.get_fq_table_name()} s
+            on d.country = s.country and similarity > 90
+
+    where status = 'currently_exporting'
```
Comment (marcuspp) [inline app/etl/tasks/datahub_country_export.py]: Maybe make this status' a constant.

File: app/db/models/external.py (hunk 1 above)
Comment (dvoong) [reply app/db/models/external.py]: done

File: app/etl/tasks/datahub_country_export.py (hunk 2 above)
Comment (dvoong) [reply app/etl/tasks/datahub_country_export.py]: done

**Pull request merged**